libr/core/cmd_type.c

```diff
@@ -583,22 +583,29 @@ static int cmd_type(void *data, const char *input) {
 		cmd_type_noreturn (core, input + 1);
 		break;
 	// t [typename] - show given type in C syntax
-	case 'u': // "tu"
+	case 'u': { // "tu"
+		char *tmp = strdup (input);
+		char *name = r_str_trim (strchr (tmp, ' '));
 		switch (input[1]) {
 		case '?':
 			r_core_cmd_help (core, help_msg_tu);
 			break;
 		case '*':
-			showFormat (core, input + 2, 1);
+			if (name != NULL) {
```

Comment:
`if (name) ` is enough , no need for `!= NULL` check , rest of it lgtm 

let's wait for tarvis !